subsys/testsuite/include/tc_util.h

```diff
@@ -62,41 +70,69 @@ static inline const char *TC_RESULT_TO_STR(int result)
 {
 	switch (result) {
 	case TC_PASS:
+		#ifndef TC_PASS_STR
 		return "PASS";
+		#else
+		return TC_PASS_STR;
+		#endif
```

Comment:
Refactor so that it is just

return TC_PASS_STR;

inside of the case statement.